packages/web/example.env

```
@@ -1,6 +1,6 @@
 VITE_NO_DEBUG_SWITCH = false
 VITE_DATA_DIR_PATH = ./public/data/programs
-VITE_DEPLOY_URL = https://gov-aid-tree.osc-fr1.scalingo.io
+VITE_DEPLOY_URL = https://tee-fronted.osc-fr1.scalingo.io
```

```suggestion
VITE_DEPLOY_URL = https://tee-frontend.osc-fr1.scalingo.io
```

Missing `n` in frontend ? :)